internal/operands/common-templates/reconcile.go

```diff
@@ -87,6 +95,32 @@ func (c *commonTemplates) Cleanup(request *common.Request) error {
 	return nil
 }
 
+func pauseCRs(request *common.Request) {
```

Comment:
I would say that this function needs to return an `error` if something went wrong with listing or patching